node/src/Producer.ts

```diff
@@ -498,9 +498,24 @@ export class Producer<ProducerAppData extends AppData = AppData>
 			throw new TypeError('every type must be a string');
 		}
 
+		// Convert event types.
+		const fbsEventTypes: FbsProducer.TraceEventType[] = [];
+
+		for (const eventType of types)
+		{
+			try
+			{
+				fbsEventTypes.push(producerTraceEventTypeToFbs(eventType));
+			}
+			catch (error)
+			{
+				// Ignore invalid event types.
+			}
+		}
+
 		/* Build Request. */
 		const requestOffset = new FbsProducer.EnableTraceEventRequestT(
-			types
+			fbsEventTypes
```

Comment:
Do we need `fbs` and `Fbs` prefixes in TypeScript? I thought everything was already converted here.

Reply:
I'm enhancing the FBS schemas on the way an creating new types when needed where there were mere strings before. So they need conversion too.